### ISSUE TITLE
Intergrate media uploader like Wordpress

### ISSUE THREAD

**Issue by phucdh** [hoangphucd3/techlogging#41]
After some researching, I found that Rails did not have any regular media uploader yet. So maybe we can use approach of Wordpress for this issue.

**Comment by hoangphucd3** [comment]
After some trying, I still can not apply Wordpress Media Management approach. There is lack of skills and knowledge about front-end side, which is based on Backbone, underscore.
For example, uploader send multiple request for just one image when uploading and I still can not figure out how to solve it.

May be this Media Manager is the next choice:
https://github.com/ctf0/Laravel-Media-Manager

**Comment by hoangphucd3** [comment]
For the last stand, I've tried to use Wordpress itself to use my API to see if it can render image fetched from server. And the result make me a little surprise, Wordpress can not render the image either, it lead to a hypothesis that the problem might place in the data responded by the server.

After some struggling, I've recognized there is some differences between the response result of Wordpress and Rails. And I found that this variable `Transfer-encoding: chunked` in Rails's response header is the source of problem. 

From this, I set this variable to another value it it works finally.
